_includes/pbc/all/install-features/202505.0/ssp/install-ssp-dashboard-management-features.md

```diff
@@ -0,0 +1,351 @@
+ssp-news-block-1,News Banner-1,Title and Content,@CmsBlock/template/title_and_content_block.twig,1,"<h5 class='spacing-x'>{{ 'ssp_dashboard.general.news' | trans }}</h5>","<h5 class='spacing-x'>{{ 'ssp_dashboard.general.news' | trans }}</h5>","<div class='grid grid--with-gutter'><div class='col col--with-gutter col--md-6 col--sm-12'>{{ content_banner('ssp-br-1', 'bottom-title') }}</div><div class='col col--with-gutter col--md-6 col--sm-12'>{{ content_banner('ssp-br-2', 'bottom-title') }}</div></div>","<div class='grid grid--with-gutter'><div class='col col--with-gutter col--md-6 col--sm-12'>{{ content_banner('ssp-br-1', 'bottom-title') }}</div><div class='col col--with-gutter col--md-6 col--sm-12'>{{ content_banner('ssp-br-2', 'bottom-title') }}</div></div>",,,,
+```
+
+</details>
+
+
+4. Append `cms_block_store.csv`:
+```csv
+ssp-news-block-1,DE
+cms-sales_rep:default,DE
+```
+5. Append `cms_slot.csv`:
+```csv
+ssp-news,ssp-news-block-1,"SSP News.",SprykerCmsSlotBlock,@SspDashboardManagement/views/dashboard/dashboard.twig,1
+```
+6. Append `cms_slot_block.csv`:
+```csv
+slt-mobile-header,blck-9,1,@ShopUi/templates/page-layout-main/page-layout-main.twig,,,,,,,
+ssp-news,ssp-news-block-1,1,@SspDashboardManagement/views/dashboard/dashboard.twig,,,,,,,
+```
+7. Append `cms_slot_template.csv`:
+```csv
+SSP Dashboard,Dashboard Page.,@SspDashboardManagement/views/dashboard/dashboard.twig
+```
+8. Append `content_banner.csv`:
+```csv
+ssp-br-1,SSP Banner Name 1,SSP Banner Description 1, ,,, ,,,/assets/current/default/images/400x200.png,,,/en/demo-landing-page,,,ssp-banner-image,,
+ssp-br-2,SSP Banner Name 2,SSP Banner Description 2, ,,, ,,,/assets/current/default/images/400x200.png,,,/en/demo-landing-page,,,ssp-banner-image,,
+```
+
+## Add twig template
+
+1. Create the file: 
```

Comment:
@supproduction  Describe what this file does

Reply:
done